### ISSUE TITLE
Lab Report 4 Feedback

### ISSUE THREAD

**Issue by shad674** [michaelchan77/cse15l-lab-reports#4]
Here is your feedback for Lab Report 4:

- [x] Provides link to their markdown-parse repository 
- [x] Provides link to the markdown-parse repository they reviewed

Snippet 1

- [ ] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation 
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program

Snippet 2

- [ ] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation 
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program

Snippet 3

- [ ] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation 
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program

Additional comments from your grader:

Feel free to comment below any questions about the rubric/feedback

Decide on what it should produce by using either VScode preview or [the CommonMark demo site](https://spec.commonmark.org/dingus/) <- take screenshots of expected outputs.
--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by michaelchan77** [comment]
Hello, in my original submission I inserted a screenshot of the section of MarkdownParseTest.java with tests for Snippets 1-3. Each test included a list called expected with the expected output corresponding to that Snippet. 
I am wondering what I need to change to meet the requirements for the checkboxes I missed? Would I need to insert screenshots of the expected output from the CommonMark demo site/VS Code preview?

Here is the link to the screenshot:
https://github.com/michaelchan77/cse15l-lab-reports/blob/main/W8_1.png

**Comment by shad674** [comment]
Yup, You just need to include a screenshot of either coomonMark or Vs Code preview, this is to know that what the expected output is what you expected it to be(evidence)

**Comment by shad674** [comment]
good job! I fixed your grade
